app/models/match_decisions/provider_only/hsa_accepts_client.rb

```diff
@@ -94,9 +94,9 @@ def declined
         match.confirm_hsa_accepts_client_decline_dnd_staff_decision.initialize_decision!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.